index.js

```diff
@@ -114,7 +114,7 @@ module.exports = {
       }).toObject(function(err, result) {
         if(result) {
           if(result.urlType == 'internal' && result._newPage) {
-            return req.res.redirect(result._newPage._url);
+            return req.res.redirect(result._newPage.slug);
```

Comment:
This

Reply:
I know you need this ASAP but this is the wrong fix.

Let me look at how you can do a redirect that takes into account that _url is already fully baked.